database_admin/migrations/018_caches_fix.up.sql

```diff
@@ -0,0 +1,114 @@
+CREATE OR REPLACE FUNCTION mark_stale_systems()
```

Comment:
Shouldn't we add previous method version into down.sql script?

Reply:
Previous version had a significant mistake, I don't see a reason to have it around.